interfaces/builtin/snap_fde.go

```diff
@@ -0,0 +1,40 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const snapFDESummary = `allows access to snapd's system-volumes API`
```

Comment:
isn't the plan to be a bit more specific than that using the byActionAcess stuff? shouldn't it says the "FDE subset of the" ?

also I though we agreed to call this snap-fde-control in the end?

Reply:
updated, thanks! for some reason it was snap-fde in my notes